kubernetes/RemoteResource/resource.yaml

```diff
@@ -233,13 +233,26 @@ items:
                             oneOf:
                               - required: [url]
                               - required: [uri]
+                              - required: [git]
                             properties:
                               url:
                                 type: string
                                 format: uri
                               uri:
                                 type: string
                                 format: uri
+                              git: 
+                                type: object
+                                required: [gittype, repo, branch, pattern]
+                                properties:
+                                  gittype:
+                                    type: string
```

Comment:
add the `enum` property so we can control what the user provides. for now i think it will just be `enum: [github]`
[example in MTP](https://github.com/razee-io/MustacheTemplate/blob/master/kubernetes/MustacheTemplate/resource.yaml#L248)